modules/unsupported/mbtiles/src/test/java/org/geotools/mbtiles/VectorLayersMetadataTest.java

```diff
@@ -0,0 +1,45 @@
+/*
+ *    GeoTools - The Open Source Java GIS Toolkit
+ *    http://geotools.org
+ *
+ *    (C) 2019, Open Source Geospatial Foundation (OSGeo)
+ *
+ *    This library is free software; you can redistribute it and/or
+ *    modify it under the terms of the GNU Lesser General Public
+ *    License as published by the Free Software Foundation;
+ *    version 2.1 of the License.
+ *
+ *    This library is distributed in the hope that it will be useful,
+ *    but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ *    Lesser General Public License for more details.
+ */
+package org.geotools.mbtiles;
+
+import static org.junit.Assert.assertEquals;
+
+import java.util.List;
+import org.apache.commons.io.IOUtils;
+import org.junit.Test;
+
+public class VectorLayersMetadataTest {
+
+    @Test
+    public void testParseMetadata() throws Exception {
+        String json = IOUtils.toString(getClass().getResourceAsStream("vectorLayers.json"));
```

Comment:
toString(java.io.InputStream) usage is deprecated.  IOUtils.toString(inputStream, StandardCharsets.UTF_8.name()) is recommended.  This is not critical, just seems like is causing a build issue on some travis build.

Reply:
Indeed, fixed it.